source/type/pb_converter.h

```diff
@@ -180,11 +180,165 @@ inline pbnavitia::CommercialMode* get_sub_object(const nt::CommercialMode*,
 inline pbnavitia::Trip* get_sub_object(const nt::MetaVehicleJourney*,
                                        pbnavitia::PtObject* pt_object) { return pt_object->mutable_trip(); }
 
+namespace {
+
+template<typename Pb> struct type_trait;
+template<> struct type_trait<nt::Line> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::Line>*  type;
+};
+template<> struct type_trait<nt::ValidityPattern> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::ValidityPattern>*  type;
+};
+template<> struct type_trait<nt::StopArea> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::StopArea>*  type;
+};
+template<> struct type_trait<nt::StopPoint> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::StopPoint>*  type;
+};
+template<> struct type_trait<nt::Network> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::Network>*  type;
+};
+template<> struct type_trait<nt::Route> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::Route>*  type;
+};
+template<> struct type_trait<nt::Company> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::Company>*  type;
+};
+template<> struct type_trait<nt::CommercialMode> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::CommercialMode>*  type;
+};
+template<> struct type_trait<nt::PhysicalMode> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::PhysicalMode>*  type;
+};
+template<> struct type_trait<nt::LineGroup> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::LineGroup>*  type;
+};
+template<> struct type_trait<ng::POI> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::Poi>*  type;
+};
+template<> struct type_trait<ng::POIType> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::PoiType>*  type;
+};
+template<> struct type_trait<nt::VehicleJourney> {
+    typedef ::google::protobuf::RepeatedPtrField<pbnavitia::VehicleJourney>*  type;
+};
+template<> struct type_trait<nt::Calendar> {
```

Comment:
I don't know if it would be useful but wouldn't it be more generic to just associate the protobuf type to the navitia type and then wrap the whole stuff with `repeatedPtrField` ?

eg (not sure it compiles and the names can be better :wink:):
```
template<> struct pb_type<nt::VehicleJourney> {
    typedef pbnavitia::VehicleJourney type;
};

template <typename T>
using pb_repeated = ::google::protobuf::RepeatedPtrField;

template<> typename pb_repeated<pb_type<nt::Line>::type>* get_mutable<nt::Line>(pbnavitia::Response& resp){
    return resp.mutable_lines();
}
```
I'm not sure the templated namespace is the best here, maybe a classic template is better:
like:
```
template<typename T> struct pb_repeated<T> {
    typedef ::google::protobuf::RepeatedPtrField<pb_type<T>::type>*  type;
};
```